README.md

```diff
@@ -7,10 +7,12 @@ This repository collects configurations of simulation codes on public compute re
 ## Importing computers into AiiDA
 
 ```
-verdi computer setup --config https:///..../mycomputer.yml
-verdi computer configure ssh mycomputer
+verdi computer setup --config https:///..../computer-setup.yml
+verdi computer configure ssh <computer-name> --config https:///..../computer-configure.yml
 ```
 
+Or promptly setting the computer configure if the `computer-configure.yml` is not provided.
```

Comment:
I suggest to put this *before* the `configure` command. I.e.

```
verdi computer setup ...
```
Some computers also require specific configuration options and provide a specific `computer-configure.yml` file:
```
verdi computer configure ssh <computer-name> --config https:///..../computer-configure.yml
```

Reply:
Sure, changed.